CHANGELOG.md

```diff
@@ -2,17 +2,23 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
-## [v0.22.0](https://github.com/puppetlabs/puppet_litmus/tree/v0.22.0) (2021-01-21)
+## [v0.23.0](https://github.com/puppetlabs/puppet_litmus/tree/v0.23.0) (2021-02-01)
 
-[Full Changelog](https://github.com/puppetlabs/puppet_litmus/compare/v0.21.0...v0.22.0)
+[Full Changelog](https://github.com/puppetlabs/puppet_litmus/compare/v0.21.0...v0.23.0)
```

Comment:
something went wrong here - why is there no `0.22.0` entry? Maybe I did screw up the tag of the last one?